hostinfo/hostinfo_linux.go

```diff
@@ -168,3 +175,72 @@ func packageTypeLinux() string {
 	}
 	return ""
 }
+
+func linuxFW() *tailcfg.LinuxFW {
+	ret := &tailcfg.LinuxFW{
+		BinInfo: make(map[string]tailcfg.LinuxFWBinInfo),
+	}
+
+	n, err := linuxfw.DetectIptables()
```

Comment:
why don't we put all this detection in its own package outside of hostinfo and have it register itself with the hostinfo package in its init?

then only tailscaled can empty-import it for its side effects.